cypress/support/step-definitions/common-steps.js

```diff
@@ -483,3 +457,19 @@ When('I click onto root in Test directory in iFrame', () => {
 Then('totalRecords is set to {string} {word}', (totalRecords, element) => {
   pagerSummary().invoke('text').should('contain', `${totalRecords} ${element}`);
 });
+
+Then('{string} component is visible', (component) => {
+  getComponentNoIframe(component).should('exist');
+});
+
+Then('{string} element is visible', (element) => {
+  getElementNoIframe(element).should('exist');
+});
+
+Then('{string} element is visible by name', (element) => {
+  getElementNoIframeByName(element).should('exist');
```

Comment:
should be `visible` or `exist` ? Please change the name of step or assertion

Reply:
deleted